uservice_ccutter/plugins/projecttypes/generic.py

```diff
@@ -2,7 +2,34 @@
 """
 import datetime
 
+from apikit import BackendError
+
+
 
 def current_year():
     """Return current year as string."""
     return str(datetime.datetime.now().year)
+  
+
+def raise_ise(text):
```

Comment:
It's in _get_travis_key, for instance.  Basically, there are lots of places where what I want to do is raise a backend error with status code 500 Internal Server Error, and this makes that convenient.